test/integration/test_groundlight.py

```diff
@@ -164,7 +164,7 @@ def test_get_detector_by_name(gl: Groundlight, detector: Detector):
 
 def test_submit_image_query_blocking(gl: Groundlight, detector: Detector):
     # Ask for a trivially small wait so it never has time to update, but uses the code path
-    _image_query = gl.submit_image_query(detector=detector.id, image="test/assets/dog.jpeg", wait=2)
+    _image_query = gl.submit_image_query(detector=detector.id, image="test/assets/dog.jpeg", wait=10)
```

Comment:
The comment seems to be at odds with what the test does now (10 seconds isn't really trivially small). 

Do we still need this test? What does it accomplish exactly? I find it confusing, e.g.`assert str(_image_query)` Why do we check that the image query's string repr is not an empty string? The following line checks that the return value is an instance of an ImageQuery, which should be sufficient.

Reply:
Regarding the comment - I've update this to be more representative of the new test behavior

> Do we still need this test? What does it accomplish exactly?
I think this test is useful as it tests the bare minimum codepath we expect our customers to call, unlike future tests that are more complex. 

> Why do we check that the image query's string repr is not an empty string? ....
I think it is reasonable that a customer might attempt to cast our code's output to a string, so making sure that this cast is successful is reasonable.